event_project/__openerp__.py

```diff
@@ -8,6 +7,7 @@
               "Serv. Tecnol. Avanzados - Pedro M. Baeza, "
               "Odoo Community Association (OCA)",
     "website": "http://www.antiun.com",
+    "images": [],
```

Comment:
This is intended to content some images, but not putting empty to bypass the warning.